src/prommis/leaching/leach_train.py

```diff
@@ -0,0 +1,315 @@
+r"""
+Leach Train
+===========
+
+Author: Andrew Lee
+
+The Leach Train unit model represents a series of leaching tanks in which a solid and liquid stream are contacted and
+undergo heterogeneous chemical reactions.
+
+Configuration Arguments
+-----------------------
+
+When creating an instance of a Leach Train model, the user must specify the number of tanks in train using the
+``number_of_tanks`` configuration argument. This value cannot be changed later.
```

Comment:
```suggestion
``number_of_tanks`` configuration argument.
```
I feel like this is a bit misleading. There's nothing stopping a user from changing the `number_of_tanks` in the configuration argument, right?

Reply:
You can change the value, but it will have no effect on the model and thus in effect this value cannot be changed.